components/web/components/org.wso2.carbon.apimgt.publisher.commons.ui/src/main/public/bootstrap3-editable/css/bootstrap-editable.css

```diff
@@ -0,0 +1,663 @@
+/*! X-editable - v1.5.1 
```

Comment:
This css file is already in the themes. Can't we reuse that rather than maintaining this in two places